diameter_synthesis/build_diameters.py

```diff
@@ -472,7 +472,7 @@ def _set_first_diams(morphology, diams, length):
 def diametrize_axon(
     morphology,
     main_diameter=1.0,
-    colateral_diameter=0.1,
+    collateral_diameter=0.1,
```

Comment:
There is a typo in the signature here so it's a breaking change... @arnaudon do you know if use this parameter somewhere?

Reply:
ah not very much, only in some local scripts, you can break to fix it! no problemo!

Reply:
Ok cool, thanks!